authors/apps/authentication/views.py

```diff
@@ -356,3 +363,63 @@ def post(self, request, **kwargs):
             }, status=status.HTTP_400_BAD_REQUEST)
         twitter_social_authentication = SocialLoginSignUp()
         return twitter_social_authentication.social_signup(user_info, **kwargs)
+
+
+class ReadingStats(RetrieveAPIView):
```

Comment:
Why is this view in the authentication app? Please separate concerns properly.

Reply:
Because it retrieving stats relating to a specific author.